test/plugin/scenarios/spring-3.0.x-scenario/config/expectedData.yaml

```diff
@@ -14,52 +14,130 @@
-  - segmentId: not null
-    spans:
-    - operationName: /healthCheck
-      operationId: 0
-      parentSpanId: -1
-      spanId: 0
-      spanLayer: Http
-      startTime: nq 0
-      endTime: nq 0
-      componentId: 14
-      isError: false
-      spanType: Entry
-      peer: ''
-      tags:
-      - {key: url, value: 'http://localhost:8080/spring-3.0.x-scenario/healthCheck'}
-      - {key: http.method, value: HEAD}
-      skipAnalysis: 'false'
-  - segmentId: not null
-    spans:
-    - {operationName: test.apache.skywalking.apm.testcase.spring3.component.TestComponentBean.componentMethod,
-      operationId: 0, parentSpanId: 1, spanId: 2, spanLayer: Unknown, startTime: nq
-        0, endTime: nq 0, componentId: 93, isError: false, spanType: Local, peer: '',
-      skipAnalysis: 'false'}
-    - {operationName: test.apache.skywalking.apm.testcase.spring3.dao.TestRepositoryBean.doSomeStuff,
-      operationId: 0, parentSpanId: 1, spanId: 3, spanLayer: Unknown, startTime: nq
-        0, endTime: nq 0, componentId: 93, isError: false, spanType: Local, peer: '',
-      skipAnalysis: 'false'}
-    - {operationName: test.apache.skywalking.apm.testcase.spring3.service.TestServiceBean.doSomeBusiness,
-      operationId: 0, parentSpanId: 0, spanId: 1, spanLayer: Unknown, startTime: nq
-        0, endTime: nq 0, componentId: 93, isError: false, spanType: Local, peer: '',
-      skipAnalysis: 'false'}
-    - operationName: /case/spring3
-      operationId: 0
-      parentSpanId: -1
-      spanId: 0
-      spanLayer: Http
-      startTime: nq 0
-      endTime: nq 0
-      componentId: 14
-      isError: false
-      spanType: Entry
-      peer: ''
-      tags:
-      - {key: url, value: 'http://localhost:8080/spring-3.0.x-scenario/case/spring3'}
-      - {key: http.method, value: GET}
-      skipAnalysis: 'false'
+  - serviceName: spring-3.0.x-scenario
+    segmentSize: ge 4
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: /healthCheck
+            operationId: 0
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 14
+            isError: false
+            spanType: Entry
+            peer: ''
+            skipAnalysis: false
+            tags:
+              - {key: url, value: 'http://localhost:8080/spring-3.0.x-scenario/healthCheck'}
+              - {key: http.method, value: HEAD}
+      - segmentId: not null
+        spans:
+          - operationName: /impl/requestmapping
+            operationId: 0
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 14
+            isError: false
+            spanType: Entry
+            peer: ''
+            skipAnalysis: false
+            tags:
+              - {key: url, value: 'http://localhost:8080/spring-3.0.x-scenario//impl/requestmapping'}
```

Comment:
The URL which has more '/' char looks strange, I think that `http://localhost:8080/spring-3.0.x-scenario/impl/requestmapping` prefer looks good.

Reply:
I found that this problem caused by `TestServiceBean#URL` has more  '/',   You can remove it to resolve the problem